test/functional/terminal/highlight_spec.lua

```diff
@@ -248,8 +248,21 @@ it('CursorLine and CursorColumn work in :terminal buffer in Normal mode', functi
     {1:bar fooba^r                                        }|
                                                       |
   ]])
-  -- CursorLine and CursorColumn are combined with TermCursorNC.
+end)
+
+-- Pending until TermCursor highlights are working again #31562
+pending('CursorLine and CursorColumn are combined with TermCursorNC', function()
```

Comment:
This is a test for the drawline.c terminal highlight combining code. If the terminal cursor is no longer highlighted by drawline.c, some other highlights should be added to the terminal.